### ISSUE TITLE
GET /parse/users/me doesn't return token

### ISSUE THREAD

**Issue by Tibabal** [ParsePlatform/parse-server#624]
Hi,

I am logging in an user with parse-server, then returning `user.getSessionToken()` to my client.
When using `PFUser becomeInBackground` providing token got from server, user successfully login, but `[PFUser currentUser].sessionToken` is null.

Could it be because of `GET /parse/users/me` doesn't return a sessionToken along with user ?
As seen with VERBOSE=1:

```
GET /parse/users/me { host: '...',
  accept: '*/*',
  'x-parse-session-token': 'r:27b08dbdcf8....',
  ...}
response: <A Parse user, without sessionToken, is this normal ?>
```

If that's normal, why my PFUser doesn't have sessionToken, which implies that it's not really logged (further request are sent without sessionToken) ?

Thanks

**Comment by nlutsenko** [comment]
Yup, looks like a valid bug, we indeed don't send the session token back on /users/me, but we should to maintain compatibility with Parse.com and support all SDKs.
Fix is incoming.

**Comment by Tibabal** [comment]
Thanks @nlutsenko !

Why is it linked to compatibility with Parse.com ? It just prevents becomeInbackground to work correctly, doesn't it ?

**Comment by nlutsenko** [comment]
SDKs expect certain things from Parse, and we are using Parse.com as a reference for things that exist in both locations (this might not be true in the future), so if Parse.com sends a token back on /login - this is what the SDKs would expect - meaning that we should maintain compatibility with both :P

**Comment by Tibabal** [comment]
Confirmed as fixed with #632 , thanks.
